src/Microsoft.DotNet.Web.ProjectTemplates/RazorPagesWeb-CSharp.csproj.in

```diff
@@ -8,7 +8,9 @@
     <WebProject_DirectoryAccessLevelKey Condition="'$(OrganizationalAuth)' == 'True' AND '$(OrgReadAccess)' != 'True'">0</WebProject_DirectoryAccessLevelKey>
     <WebProject_DirectoryAccessLevelKey Condition="'$(OrganizationalAuth)' == 'True' AND '$(OrgReadAccess)' == 'True'">1</WebProject_DirectoryAccessLevelKey>
   </PropertyGroup>
+  <!--#if (IndividualLocalAuth && UseLocalDB) -->
```

Comment:
What is this empty region for?

Reply:
Why not have an empty line always in between `PropertyGroup` and `ItemGroup` here?

Reply:
The `ItemGroup` right below only applies for certain conditions - using IndividualLocalAuth and if UseLocalDB is false. So when we add the ItemGroup we add the space.